sdk-extensions/view-config/README.md

```diff
@@ -0,0 +1,63 @@
+# OpenTelemetry View Config SDK Extension
+
+[![Javadocs][javadoc-image]][javadoc-url]
+
+This artifact contains experimental code for enabling file based YAML configuration of Metric SDK Views.
+
+For example, a YAML file at `/Users/user123/view.yaml` could be used to configure views by calling:
+
+```
+SdkMeterProviderBuilder builder = SdkMeterProvider.builder();
+ViewConfig.registerViews(builder, new File("/Users/user123/view.yaml"));
+```
+
+Suppose `/Users/user123/view.yaml` had content as follows:
+
+```yaml
+- selector:
+    instrumentName: my-instrument
+    instrumentType: COUNTER
+    meterName: my-meter
+    meterVersion: 1.0.0
+    meterSchemaUrl: http://example.com
+  view:
```

Comment:
I love this!

1. we may want to think about providing "default" views for our instrumentation APIs as well, regarding passing ALL attributes to a meter and reducing down to the semantic conventions).
2. We'd like to (eventually) have SDK configuration components be accessible via config, similar to this.   Are you planning to use this implementation to propose addition to the Specification?  I think you should as I'd like for some consistency in SDK config across languages.
3. Regarding "precedence" order, I assume it's just the order these files are passed into `registerViews`? 
4. Should we offer any "logging-like" configuration-lookups via things like ServiceLoader?

Reply:
> 1. we may want to think about providing "default" views for our instrumentation APIs as well, regarding passing ALL attributes to a meter and reducing down to the semantic conventions).

Yes, this could be extended to include additional parameters which translate to the attribute / baggage components of the view. For example, could have `attributeFilterAllowableKeys: ["foo", "bar", "baz"]` to remove attribute not in the allow list.

>2. We'd like to (eventually) have SDK configuration components be accessible via config, similar to this. Are you planning to use this implementation to propose addition to the Specification? I think you should as I'd like for some consistency in SDK config across languages.

Yeah I like the idea for a file based configuration specification for the SDK. Configuration of the view API is a pretty urgent gap IMO, so I figured its a nice place to build prototypes that can be used to explore ideas and collect feedback. I definitely think that having a consistent spec for file based config that crosses language is important. 

>3. Regarding "precedence" order, I assume it's just the order these files are passed into registerViews?

Yes, if you were to call this with multiple files, the views would be registered in order in which the files were parsed, and for each file, in the order that the views were configured.

>4. Should we offer any "logging-like" configuration-lookups via things like ServiceLoader?

Do you mean like searching for configuration files in standard locations? If so, I'm not opposed, but personally would prefer to have an environment variable point to the absolute path of a config file.